interface/forms_admin/forms_admin.php

```diff
@@ -181,8 +181,6 @@
     {
             // added 8-2009 by BM - do not show the metric vitals form as option since deprecated
```

Comment:
Remove the comments related to this code as well then it will be ready for merge.